### PR TITLE
Add support for custom arguments 

### LINKED ISSUE #21: [TASK] Add the ability to pass custom arguments to the plugin from a json file
## Task Description and Info (what needs to be done?)

The user might want to pass custom arguments from to their written plugin. There is no current support for this in GridGopher, however, adding this feature is simple and straightforward.

## Steps and Acceptance Criteria (when is it considered complete?)

- [ ] 1. A CLI argument can accept the path to a json file that contains a variety of values that will be passed as arguments
- [ ] 2. `main.py` should accept that optional argument, open and read the json file
- [ ] 3. pass the content of the json file if an argument was passed to the plugins' run function
- [ ] 4. a `**kwagrs` should be included in the run function of the plugin

## Time to complete (approximately, how much time will this take?)

Add estimate here

## Person/People involved (who is working on this?)

@

## Other information

Anything else that doesn't fit in previous categories

### PR DESCRIPTION
**Pull request by tuduun**
This pull request adds the ability to read a JSON file and pass the contents/arguments into a plugin. It also adds a command-line option to accept the file's path. 

closes #21

### CODE REVIEW

File: sheetshuttle/main.py [hunk 1]
```diff
@@ -59,5 +68,14 @@ def load_plugin(directory: str, name: str):
     return plugin_source, my_plugin
 
 
+def load_json_file(file_path):
+    """Return the contents of a json file in the file path."""
+    if not file_path:
+        return {}
+    with open(file_path, encoding="uts-8") as read_file:
+        data = json.load(read_file)
+        return data
```
Comment (noorbuchi) [inline sheetshuttle/main.py:77]: We should handle the error where the file does not exist and raise that error with a message. Please add that in a `try` statement

**Review by noorbuchi — CHANGES_REQUESTED** [review]

**Review by antlet — CHANGES_REQUESTED** [review]
Once the error handling is implemented like Noor mentioned, everything looks good to go and I will approve.

**Comment by noorbuchi** [comment]
I ended up fixing some of the issues we discussed in addition to a bug that I remembered. Further testing is needed on some aspects but I think we can merge it for now. Can you please review this @antlet @tuduun @Yanqiao4396

**Review by noorbuchi — APPROVED** [review]

**Review by antlet — APPROVED** [review]
Looks good!

**Pull request merged**